modules/ROOT/pages/6.3-release-notes.adoc

```diff
@@ -0,0 +1,697 @@
+
+
+=== Dragging a noneditable element before or after another noneditable element now works correctly.
+
+Elements that had the `contenteditable="false"` attribute set could not be dragged to the left of an ancestor element.
+
+For example in the following
+
+[source,html]
+----
+<p>Text in a paragraph tag with a span of <span contenteditable="false">non-editable text</span> inside.</p>
+----
+
+the string _non-editable text_ could not be dragged to before the opening `+<p>+` tag.
+
+{productname} 6.3 can place a fake insertion point next to elements with a `contenteditable="false"` attribute when another element with a `contenteditable="false"` attribute is dragged, allowing the object to be moved before and after other such elements.
+
+
+=== The restored selection or location after a redo or undo action was not scrolled into view.
+
+Prior to the {productname} 6.3 release, executing an undo or redo function in the editor did not scroll the editor to the new insertion point or selection, should the undo or redo result in the insertion point or selection being out of the current viewport.
+
+Instead, users had to manually scroll to the insertion point or selection after issuing an undo or redo command that resulted in the insertion point or selection being out of view.
+
+As of the {productname} 6.3 release, the scroll function now repositions the insertion point to the last known undo or redo location.
+
+
+=== A newline could not be inserted after an inline element with a contenteditable="false" attribute was created.
+
+In previous versions of {productname}, a newline could not be inserted when the selection was restored from a ‘bookmark’ positioning after adding an inline element with a `contenteditable="false"` attribute.
+
+This was most commonly seen by pressing the *Return* (or *Enter*) key not moving the insertion point to a new line after a merge tag was created in the editor.
+
+For {productname} 6.3, a fix was applied to the core `SelectionOverrides` settings to ensure the user selection restores from the ‘bookmark’ positioning before and after an inline element with a `contenteditable="false"` attribute is created.
+
+Hitting the *Return* (or *Enter*) after creating a merge tag now moves the insertion point to a new line, as expected.
+
+
+=== Inline mode did not respect the `content_css_cors` option.
+
+In previous versions of {productname}, the `tinymce.styleSheetLoader` global was not configured by the `content_css_cors` option, when specified.
+
+Consequently, stylesheets loaded by {productname} onto the outer document page did not have the `crossOrigin="anonymous"` attribute. This particularly effected {productname} instances running in inline mode, because the `tinymce.styleSheetLoader` global is used to load stylesheets used by inline editors. (Stylesheets for {productname} editors running in iframe mode are specific to the editor.)
```

Comment:
```suggestion
Consequently, stylesheets loaded by {productname} onto the outer document page did not have the `crossOrigin="anonymous"` attribute. This particularly affected {productname} instances running in inline mode, because the `tinymce.styleSheetLoader` global is used to load stylesheets used by inline editors. (Stylesheets for {productname} editors running in iframe mode are specific to the editor.)
```